docs/requirement-sets/common/image-coercion-requirement-sets.md

```diff
@@ -16,20 +16,20 @@ ImageCoercion 1.1 enables conversion to an image (`Office.CoercionType.Image`) w
 
 - Excel on Windows
   - Microsoft 365 subscription
-  - perpetual Office 2016 and later
+  - perpetual Office 2021 and later
```

Comment:
Again, here, why make this change and imply something false? Just because _we_ don't support Office 2016 doesn't mean that _it_ doesn't support **ImageCoercion 1.1**.